presto-main/src/main/java/com/facebook/presto/sql/gen/BytecodeUtils.java

```diff
@@ -43,9 +43,6 @@
 import static com.facebook.presto.bytecode.expression.BytecodeExpressions.constantTrue;
 import static com.facebook.presto.bytecode.expression.BytecodeExpressions.invokeDynamic;
 import static com.facebook.presto.operator.scalar.ScalarFunctionImplementation.ArgumentType.VALUE_TYPE;
-import static com.facebook.presto.operator.scalar.ScalarFunctionImplementation.NullConvention.RETURN_NULL_ON_NULL;
-import static com.facebook.presto.operator.scalar.ScalarFunctionImplementation.NullConvention.USE_BOXED_TYPE;
-import static com.facebook.presto.operator.scalar.ScalarFunctionImplementation.NullConvention.USE_NULL_FLAG;
```

Comment:
how is that possible that build did not fail here? Maybe there is a bug in code style check?

Reply:
Checkstyle didn't catch them as unused, because the enum names are used in a switch statement (of course this doesn't require an import, but seems enough to mislead the checkstyle)